CMakeLists.txt

```diff
@@ -272,6 +272,9 @@ option(Slicer_USE_FOLDERS "Organize build targets into folders" ON)
 mark_as_superbuild(Slicer_USE_FOLDERS)
 mark_as_advanced(Slicer_USE_FOLDERS)
 
+option(Slicer_USE_SYSTEM_vtkAddon "Build Slicer using system vtkAddon" OFF)
+mark_as_superbuild(Slicer_USE_SYSTEM_vtkAddon:BOOL)
```

Comment:
To be very specific, we should support `vtkAddon_DIR` instead of `Slicer_USE_SYSTEM_vtkAddon`